src/amount.h

```diff
@@ -28,7 +28,7 @@ extern const std::string CURRENCY_UNIT;
  * critical; in unusual circumstances like a(nother) overflow bug that allowed
  * for the creation of coins out of thin air modification could lead to a fork.
  * */
-static const CAmount MAX_MONEY = 21000000 * COIN;
+static const CAmount MAX_MONEY = 30500000 * COIN;
```

Comment:
Zcoin going to have 30.5M coins?

Reply:
what? this is new to me!

Reply:
@reubenyap @a-bezrukov @psolstice

Reply:
switch to MTP will halve rewards per block because the distance between blocks will reduce to 5 minutes. We changed the bitcoin's subsidy halving block number (originally 210000) to keep total reward amount over given period of time exactly the same. This is a fix for bitcoin test, nothing more. Incorrectly named.